README.md

```diff
@@ -1,8 +1,9 @@
-# Tealium Mobile Library for Android Version 5.0.4
+# Tealium Mobile Library for Android
 
```

Comment:
Can optionally add following to get Github widgets like we have on Swift:

[![License](https://img.shields.io/badge/license-Proprietary-blue.svg?style=flat
           )](https://github.com/Tealium/tealium-android/blob/master/LICENSE.txt)
[![Platform](https://img.shields.io/badge/platform-Android-lightgrey.svg?style=flat
             )](https://developer.apple.com/resources/)
[![Language](http://img.shields.io/badge/language-Java-orange.svg?style=flat
             )](https://developer.android.com)